router/router.go

```diff
@@ -175,12 +175,22 @@ func New(cfg *config.Configuration, rateConvertor *currencies.RateConverter) (r
 	r = &Router{
 		Router: httprouter.New(),
 	}
+
+	// For bid processing, we need both the hardcoded certificates and the certificates found in container's
+	// local file system
+	certPool := ssl.GetRootCAPool()
+	var readCertErr Error
+	certPool, readCertErr = ssl.AppendPEMFileToRootCAPool(certPool, cfg.PemCertsFile)
```

Comment:
Do we need a check if the string is empty?

Reply:
The `AppendPEMFileToRootCAPool(certPool *x509.CertPool, pemFileName string)` function checks for an empty string but I removed the error it was returning when the string is empty because as of right now, we have no certificates file and it seems odd to return an error if the certificates file has not been defined in the configuration.

Do you prefer the calling function to do the empty string check?